CODEOWNERS

```diff
@@ -4,32 +4,55 @@
 
 distribution/ddf-common/*	@stustison
 
+distribution/docs/*         @ricklarsen
+
 
 ###############################
 # CATALOG
 ###############################
 
+catalog/async/*   @peterhuffer  @emmberk
+
+catalog/core/catalog-core-api/*             @brendan-hofmann
```

Comment:
I looked around and after refreshing my memory on all that's in there, I too agree that's pretty broad. @brendan-hofmann let's work to refine what you're interested in.